### DISCUSSION TITLE
[Unstable] abandoned outpost mission on mission mode can take you to the Eye

### DISCUSSION THREAD

**Discussion by EebleSheembl** [FakeFishGames/Barotrauma#15275] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Went onto mission mode to do an abandoned outpost, got a mission to go to the Eye

(Mod was Combat Overhaul Legacy which I don't think would do anything to the Eye mission)

### Reproduction steps

Start Server
Go onto mission mode
Select abandoned outpost

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by EebleSheembl** [comment]
Tested it some more without COL and its only happening with the mod, what the fuck

**Comment by Regalis11** [comment]
Thank you for the report!

I gave this a test with the "Combat Overhaul Legacy" mod enabled, and I'm seeing slightly different results: when I select the Abandoned Outpost mission type, I instead get the "Rescuing Subra" mission. I think this happens because the mod seems to override all the abandoned outpost missions and set their types as "Destroy outpost", which only leaves that one specific mission with the generic "Abandoned outpost" type.

Which specific abandoned outpost mission type had you selected? Do you have any other mods installed that might modify the missions? Which seed did this happen on?

**Comment by Ek-Videogames** [comment]
I think this interaction is somehow related to how the Eye of Europa mission locations are technically treated as "outposts modules".

It can be caused by various reasons like an outpost generating in the wrong spot on the biome map, or mods that change how the campaign map is structured. When game for whatever reason "cannot find suitable module for the location type" it wrongwarps you to the blank end-zone biome gate.

I'm guessing this "combat enhanced" mod does something to overwrite outpost modules, or the outpost mission 'module tags'. Which would be what caused this.
